config/routes.rb

```diff
@@ -24,6 +24,7 @@
 
   resources :shipments
   resources :user_invitations, path: "/users/invitations", only: [:new, :create, :index, :show, :update]
+  resources :order_notes
```

Comment:
Can you add an `only: [...]` with the actions you are using? I've found it helpful to be precise about the routes so we don't have excess routes that aren't in use, and to know what might be affected when making changes later, or to make navigating the codebase a bit easier later.